Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -312,9 +312,11 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
   }
 
   for (const auto [bottom, middle, top] : spacePointsGrouping) {
-    m_seedFinder.createSeedsForGroup(
-        m_cfg.seedFinderOptions, state, spacePointsGrouping.grid(),
-        std::back_inserter(seeds), bottom, middle, top, rMiddleSPRange);
+    VectorPolicy out_policy_cont(seeds);
+    GenericBackInserter back_inserter(out_policy_cont);
+    m_seedFinder.createSeedsForGroup(m_cfg.seedFinderOptions, state,
+                                     spacePointsGrouping.grid(), back_inserter,
+                                     bottom, middle, top, rMiddleSPRange);
```

Comment:
```suggestion
    VectorPolicy outPolicyContainer(seeds);
    GenericBackInserter backInserter(outPolicyContainer);
    m_seedFinder.createSeedsForGroup(m_cfg.seedFinderOptions, state,
                                     spacePointsGrouping.grid(), backInserter,
                                     bottom, middle, top, rMiddleSPRange);
```
These sections of ACTS do not really align with our [naming convention](https://acts.readthedocs.io/en/v34.0.0/codeguide.html#naming), but I had the feeling that new parts were usually added in ACTS-style. Maybe you could adapt this (for YOUR changes only) and only keep the templated paramaters like `template <typename sp_range_t>` in snake_case.

Reply:
It should be fine now, thanks!